<launch>

<arg name="imu_port" />
<param name="imu_port" value="$(arg imu_port)" />

<arg name="gps_port"/>
<param name="gps_port" value="$(arg gps_port)" />

<node pkg="lab4_driver" type="imu_driver.py" name="imu_driver" output="screen">
</node>

<node pkg="lab4_driver" type="euler_to_quaternion_server.py" name="euler_to_quaternion_server" output="screen">
</node>

<node pkg="lab4_driver" type="gps_driver.py" name="gps_data" output="screen">
</node>

</launch>
